getters.go

```diff
@@ -128,16 +141,31 @@ func (ko *Koanf) MustInt(path string) int {
 // empty []int slice if the path does not exist or if the value
 // is not a valid int slice.
 func (ko *Koanf) Ints(path string) []int {
-	ints := ko.Int64s(path)
-	if len(ints) == 0 {
+	o := ko.Get(path)
+	if o == nil {
 		return []int{}
 	}
 
-	out := make([]int, len(ints))
-	for i, v := range ints {
-		out[i] = int(v)
+	var out []int
+	switch v := o.(type) {
+	case []int:
+		return v
+	case []interface{}:
```

Comment:
Shouldn't there be a case for `[]int64` as well? A slice loaded from JSON would be of that type, right?

Similarly in `Ints()` also, right?

Reply:
A slice loaded from JSON would be `[]float64`, a slice loaded from YAML would be `[]int`.
But you're right, previously we did support it. And if it's a `[]interface` the consists of `int64` it will be converted.
Adding it.